src/customJs/commonProperties/utils.js

```diff
@@ -0,0 +1,15 @@
+export const formatUrl = (valueWithOutFormat) => {
+  if (!valueWithOutFormat) {
+    return '';
+  }
+
+  const withoutBeginOrEndSpaces = valueWithOutFormat
+    .replace(/(^[ ]*)/, '')
+    .replace(/[ ]*$/, '');
+
```

Comment:
@andresmoschini  What do you think about this replace?

Reply:
Mmmm, I do not understand these regexes very well. Why not simply `.trim()`?

Reply:
Oh, I understand now

```suggestion
  const withoutBeginOrEndSpaces = valueWithOutFormat
    .replace(/(^ *)/, '')
    .replace(/ *$/, '');
```

Or better:

```suggestion
  const withoutBeginOrEndSpaces = valueWithOutFormat
    .replace(/(^\s*)/, '')
    .replace(/\s*$/, '');
```

But, I still think that `trim()` is better.

Reply:
You are right!! I will use the trim() method.